.drone/drone.sh

```diff
@@ -117,30 +123,28 @@ common_install() {
   git clone https://github.com/boostorg/boost-ci.git boost-ci-cloned --depth 1
   cp -prf boost-ci-cloned/ci .
   rm -rf boost-ci-cloned
-  if [ "$boost_cache_hit" != true ]; then
-    . ./ci/common_install.sh
-    if command -v rsync &>/dev/null && [ "$drone_cache_rebuild" == true ]; then
-      if command -v apt-get &>/dev/null; then
-        apt-get install -y rsync
-      fi
-      mkdir "$cache_dir"/boost
-      rsync -a "$BOOST_ROOT/" "$cache_dir/boost"
-      echo "$boost_cache_key" >"$cache_dir/boost_cache_key.txt"
-    fi
-  else
+  if [ "$boost_cache_hit" = true ]; then
     cd ..
     mkdir boost-root
     cd boost-root
-    BOOST_ROOT="$(pwd)"
-    export BOOST_ROOT
-    ls "$cache_dir/boost"
+    export BOOST_ROOT="$(pwd)"
     if command -v apt-get &>/dev/null; then
       apt-get install -y rsync
     fi
     rsync -a "$cache_dir/boost/" "$BOOST_ROOT"
-    rm -rf "$BOOST_ROOT/libs/$SELF"
-    rsync -a "$BOOST_CI_SRC_FOLDER/" "$BOOST_ROOT/libs/$SELF" --exclude cache
-    $python_executable tools/boostdep/depinst/depinst.py --include benchmark --include example --include examples --include tools "$SELF"
+    cd $DRONE_WORKSPACE
+  fi
+  . ./ci/common_install.sh
+
+  if [ "$drone_cache_rebuild" == true ]; then
+    if command -v apt-get &>/dev/null; then
+      apt-get install -y rsync
```

Comment:
Should the images include `rsync`? I couldn't replicate the command with `cp` but this conditional looks like a workaround.

Reply:
The images should include `rsync` but they don't at the moment. This workaround isn't too bad.